src/blocks/ProductList/product/Component.tsx

```diff
@@ -1,32 +1,136 @@
 import React from 'react'
 
-import type { EuProduct } from '@/payload-types'
+import type { Category, Country, EuProduct, ReplacedProduct, Subcategory } from '@/payload-types'
 
 import { Media } from '@/components/Media'
+import { Card, CardContent, CardDescription, CardFooter, CardHeader } from '@/components/ui/card';
+import { H2, H4 } from '@/components/ui/typography';
+import { Button } from '@/components/ui/button';
+import { Link } from '@payloadcms/ui';
 
 interface ProductCardProps {
   product: EuProduct;
 }
 
 export const ProductCard: React.FC<ProductCardProps> = async ({ product }) => {
-
+  const hasCategory = product.categories !== null && product.categories !== undefined && product.categories.length !== 0
+  const hasSubcategory = product.subcategories !== null && product.subcategories !== undefined && product.subcategories.length !== 0
+  const replacesProducts = product.replaces !== null && product.replaces !== undefined && product.replaces.length !== 0
   return (
-    <li className="container mt-16">
-      <div className="max-w-[48rem]">
-        {product.logo &&
-          <Media
-            imgClassName='h-32 w-32 border border-border rounded-[0.8rem]'
-            resource={product.logo}
-          />
-        }
-        <br />
-        Name: {product.name}
-        <br />
-        Description: {product.description ?? "No description"}
-        <br />
-        Link: {product.link ?? "No link"}
-      </div>
-    </li>
+    <>
+      <Card className='h-full flex flex-col'>
+        <CardHeader>
+          <div className='flex flex-col md:flex-row justify-between'>
+            <div className='flex items-center justify-center'>
+              <Media
+                imgClassName='w-12 h-12 border border-border rounded-full'
+                resource={product.logo}
+                alt={`${product.name} logo`}
+              />
+              <H2 className='mx-2 pb-0 flex-wrap'>
+                {product.name}
+              </H2>
+            </div>
+            <div className='flex items-center justify-center '>
+              <H4 className='pb-0'>
+                {product.producedIn?.map((c: Country | number) => {
+                  if (typeof c !== 'number') {
+                    return <p key={c.id}>{c.name}</p>
+                  }
+                }
+                )}
+              </H4>
+            </div>
+          </div>
+        </CardHeader>
+
+        <div className='flex flex-col justify-between h-full gap-2'>
+          <CardDescription className='text-ellipsis overflow-hidden h-fit p-6' >
+            {product.description}
+          </CardDescription>
+          <div className='flex flex-col'>
+            <CardContent className='flex flex-col justify-end text-xs gap-2'>
+
+              {hasCategory && (
+                <div>
```

Comment:
```suggestion
                <>
```
You can use React Fragment here as wrapper. Also below in `hasSubcategory` & `replacesProducts`